src/cli.h

```diff
@@ -0,0 +1,26 @@
+/**
+ *
+ * @file	cli.h
+ * @brief 	Header defining stuff for command line argument parsing 
+ * @author 	Moritz R. Huellmann
+ * @date 	10.10.22
+ *
+ */
+
+#ifndef _CRETRO_SRC_CLI_H_
+#define _CRETRO_SRC_CLI_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef struct config {
+    bool debug;
+    uint32_t us_delay;
+    const char* rom;
```

Comment:
Probably this should be renamed to something like `rom_name` since `rom` together with the type const char* could look like a pointer storing the raw rom bytes.

Reply:
You're absolutely right, changing it!